distribution/manager.go

```diff
@@ -12,16 +12,16 @@ type DistributionServicesManager struct {
 	config config.Config
 }
 
-func New(commonDetails *auth.CommonDetails, config config.Config) (*DistributionServicesManager, error) {
-	err := (*commonDetails).InitSsh()
+func New(ServiceDetails *auth.ServiceDetails, config config.Config) (*DistributionServicesManager, error) {
```

Comment:
I suggest changing the function argument to start with a lower case letter, to follow our conventions.